Data/Sequence/Internal.hs

```diff
@@ -385,20 +385,53 @@ fmapSeq f (Seq xs) = Seq (fmap (fmap f) xs)
  #-}
 #endif
 
+#if __GLASGOW_HASKELL__ >= 708
+(.#) :: Coercible a b => (b -> c) -> (a -> b) -> a -> c
+(.#) f _ = coerce f
+#else
+(.#) :: (b -> c) -> (a -> b) -> a -> c
+(.#) f g = \x -> f (g x)
```

Comment:
This can be written `(.#) = (.)`. Should we add a hidden `Coercions` module in `Utils` for this and `(#.)`?

Reply:
I think that'd be a good idea. Another candidate for it would be `liftLeftFold` (or something, I'm not sure of the convention) which could be instead of `lift_elem`